### DISCUSSION TITLE
Be able to delete the terminal

### DISCUSSION THREAD

**Discussion by Lchaxlover** [winstxnhdw/lc-hax#484] [Ideas]
I know someone who makes his own hacks for lc and he can do this. Can you?

**Comment by Totoqoe** [comment]
The terminal probably isn't deleted but instead, is moved far away or out of bounds.

**Reply by Lchaxlover** [reply]
Ok well can you guys do that

**Reply by winstxnhdw** [reply]
It’s host-only since v49

**Reply by Lchaxlover** [reply]
oh alr. Well is it possible to like end the game? Like fully end it no just make the ship leave.

**Reply by winstxnhdw** [reply]
Without crashing the host? No, I don’t think so.

**Reply by Lchaxlover** [reply]
No with crashing the host

**Reply by Lchaxlover** [reply]
How about being able to make maps eclipsed? without being host

**Reply by winstxnhdw** [reply]
We don’t really want to crash the host since we are in a symbiotic relationship with them anyways. And no, there’s no way to make maps eclipsed.

**Reply by Lchaxlover** [reply]
in a symbiotic relationship with them? Bro what

**Reply by Lchaxlover** [reply]
How about being able to trap people, like spawning the red thingy on em

**Reply by Lchaxlover** [reply]
Kinda like the /random but instead of  tping them it just puts the thing on em

**Reply by Lchaxlover** [reply]
Also I know you guys say that there is no way to spawn mobs in other peoples lobbies, but like 4 weeks ago there was a hacker in my game before i had hacks and he could. He was able to do a lot of host only stuff. Is there not a way you guys can bypass it with a little work?

**Reply by winstxnhdw** [reply]
What’s the point if you can’t build outside anymore? Players can just enter build mode and store the cupboard.

Also, about doing host-only stuff, he was probably the host. You can’t just do it. If the host’s code says you can’t do it, you can’t do it. This is not P2P networking. This is client-host RPC.

Of course, if you ever find a way, feel free to leave it in discussions. We have one dude that has attempted several times only to be wrong every single time. Just filter for every discussion by arandomuser0.

**Reply by Lchaxlover** [reply]
He wasnt the host thats what im trying to tell you. This hacker called "George" had been following this guy for like 7 hours straight and he would not leave him alone. He was a god he could do whatever he wanted in the other guys server.

**Reply by arandomuser0** [reply]
crazy

**Reply by winstxnhdw** [reply]
Unless I see it myself, I am not going to invest time to research into this. If this is possible, it’ll probably end up released somewhere and we’ll look into it again.
